openshift/README.md

```diff
@@ -37,7 +37,10 @@ oc create -n k8vopl-<env> configmap getok-sc-config \
   --from-literal=SC_KC_PROD_ENDPOINT=https://sso.pathfinder.gov.bc.ca \
   --from-literal=SC_KC_INT_REALM=jbd6rnxw \
   --from-literal=SC_KC_TEST_REALM=jbd6rnxw \
-  --from-literal=SC_KC_PROD_REALM=jbd6rnxw
+  --from-literal=SC_KC_PROD_REALM=jbd6rnxw \
+  --from-literal=CHES_TOKEN_ENDPOINT=https://sso-dev.pathfinder.gov.bc.ca/auth/realms/jbd6rnxw/protocol/openid-connect/token \
+  --from-literal=CHES_EMAIL_ENDPOINT=https://ches-master-9f0fbe-dev.pathfinder.gov.bc.ca/api/v1/email \
+  --from-literal=CHES_HEALTH_ENDPOINT=http://ches-master-9f0fbe-dev.pathfinder.gov.bc.ca/api/v1/health
```

Comment:
Not sure if we should be storing the entire endpoint url in these variables instead of saving a base + path. It works here since there's so few, but if this was a more complex api with many endpoints it starts making less sense in its current form.

Reply:
See comment above, would adjust if more than 2 endpoints.